client-app/ui-kit/components/organisms/add-to-cart/vc-add-to-cart.vue

```diff
@@ -87,6 +74,16 @@ const rules = computed(() => toTypedSchema(quantitySchema.value));
 
 const { errorMessage, validate, setValue } = useField("quantity", rules);
 
+async function validateFields(): Promise<void> {
+  const { valid } = await validate();
+
+  if (!valid && errorMessage.value) {
+    emit("validationError", errorMessage.value);
+  } else {
+    emit("validationError", undefined);
```

Comment:
You may just call `emit("validationError")`, but I'd rather prefer have two separate events `update:validationError` and `remove:validationError` or single `(event: "update:validation", value: { isValid: true } | { isValid: false, errorMessage: string })` to clarify why there is no error.